app/src/main/res/values/strings.xml

```diff
@@ -1037,6 +1037,10 @@
     <string name="etp_panel_on">Protections are ON for this site</string>
     <!-- Enhanced Tracking Protection message that protection is currently off for this site -->
     <string name="etp_panel_off">Protections are OFF for this site</string>
+    <!-- Text to announce when Enhanced Tracking Protection switch is ON -->
+    <string name="etp_switch_on">Enhanced Tracking Protection On</string>
```

Comment:
@asavill do you have any thoughts on changing this to not adding new strings and using the `etp_panel_on` and `etp_panel_off` strings here? Seems like they are saying very similar things?

Reply:
@ekager I think etp_panel_on/off works well, too.

### It would mean:

GIVEN: ETP toggle is enabled
WHEN: User focuses etp toggle with talkback enabled
THEN: Talkback announces "Protections are ON for this site, switch. Double-tap to toggle

GIVEN: ETP toggle is disabled
WHEN: User focuses etp toggle with talkback enabled
THEN: Talkback announces "Protections are OFF for this site, switch. Double-tap to toggle

Happy to take your lead on it.